### PR TITLE
Учебный проект: время разбивать камни

### PR DESCRIPTION
**Pull request by grigorievk**
---
:mortar_board: [Учебный проект: время разбивать камни](https://up.htmlacademy.ru/ecmascript/9/user/1075187/tasks/4)

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -3,4 +3,8 @@ env:
   browser: true
```
Comment (rnskv) [inline .eslintrc.yml]: Please look at how to prevent PR, and correct its comments in this code.

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  mode: `development`,
+  entry: `./src/main.js`,
+  output: {
+    filename: `bundle.js`,
+    path: path.join(__dirname, `public`)
+  },
+  devtool: `source-map`,
+  devServer: {
+    contentBase: path.join(__dirname, `public`),
+    publicPath: 'http://localhost:8080/',
```
Comment (rnskv) [inline webpack.config.js]: May be create some folder for public assets?

File: .eslintrc.yml (hunk 1 above)
Comment (grigorievk) [reply .eslintrc.yml]: Sorry, I didn't understand what kind of comment is in question, I was just following the lesson.

File: webpack.config.js (hunk 2 above)
Comment (grigorievk) [reply webpack.config.js]: You mean a structure like this?
|-public
|--assets
|---css
|---fons
|---img

Comment (rnskv) [reply webpack.config.js]: Yes. Or why you need this param? By default this looked at "/"

File: .eslintrc.yml (hunk 1 above)
Comment (rnskv) [reply .eslintrc.yml]: This comment means that it is necessary to correct the mistakes of the second project in this

File: webpack.config.js (hunk 2 above)
Comment (grigorievk) [reply webpack.config.js]: Thanks for pointing that out, it's mindless copypaste.

**Pull request merged**